.gitmodules

```diff
@@ -184,6 +184,10 @@
 [submodule "community-build/community-projects/protoquill"]
 	path = community-build/community-projects/protoquill
 	url = https://github.com/dotty-staging/protoquill.git
+[submodule "community-build/community-projects/Monocle"]
+	path = community-build/community-projects/Monocle
+	url = https://github.com/optics-dev/Monocle.git
+	branch = scala-3-RC2
```

Comment:
```suggestion
```

We should use the default upstream branch, not one tied to a particular release candidate.